src/injector/stage1/release.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
For consitency, let's move this to the makefile and call the makefile from the github actions as well vs having random shells scripts floating around. Also don't think we nee dot get fancy with the size reporting, a simple `ls -lah <filename>` is sufficient and almost the same result.

Reply:
o yea, this file is temporary just for local quick building of both binaries and viewing size differentials, just before approval ill remove

Reply:
* delete me plz *